core/arch/x86/x86.asm

```diff
@@ -1884,8 +1884,8 @@ GLOBAL_LABEL(cpuid_supported:)
         DECLARE_FUNC(our_cpuid)
 GLOBAL_LABEL(our_cpuid:)
         mov      REG_XAX, ARG1
-        mov      REG_XDX, ARG2
         mov      REG_XCX, ARG3
+        mov      REG_XDX, ARG2
```

Comment:
I think this should have a comment b/c it looks like a mistake with the 1,3,2 ordering.

Reply:
Added a comment.